pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,134 @@
+{ config
+, pkgs
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, python35Packages
+, stdenv
+, bash
+, intltool
+, glib
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, wrapGAppsHook
+}:
+
+python35Packages.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
+  src = fetchurl {
+    url = "https://launchpad.net/onboard/${majorVersion}/${version}/+download/${name}.tar.gz";
+    sha256 = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [
```

Comment:
I think what you want is to have some libraries available at runtime. Adding them to `propagatedBuildInputs` is not doing that. Instead, what it does is add them to the `buildInputs` of any package that depends on this package.

If your package needs certain libraries during runtime, then it needs to patch the `rpath` or otherwise wrap the executables with a wrapper that sets `LD_LIBRARY_PATH`.

Reply:
Only the python dependencies should be part of `propagatedBuildInputs`, everything else should go to `buildInputs` if it is a library we want to link against or something we want in any way to be used by the built program, `nativeBuildInputs` if it is only used during building the package (intltool wrapGAppsHook glibcLocales).